app/controllers/api/v1/workflows_controller.rb

```diff
@@ -24,6 +24,11 @@ def destroy_links
     super { |workflow| reload_queue(workflow) }
   end
 
+  def retire_subject
+    controlled_resource.retire_subject(params[:subject_id])
+    render nothing: true
```

Comment:
Nero doesn't need more than the 200, and rendering the workflow added 150ms of overhead and a bunch of queries, that's why I added this line in.